content/actions/creating-actions/creating-a-docker-container-action.md

```diff
@@ -189,6 +188,8 @@ jobs:
     steps:
     - name: Hello world action step
       id: hello
+      # For unpublished actions put <username>/<action-repository-name>@<branch-name>
+      # For published actions put <username>/<action-repository-name>@<version>
```

Comment:
```suggestion
```

Ah, sorry; I missed this in my first review!

This is a bit misleading; after the `@` can be _any_ git reference (e.g. branch names, tags, SHAs, etc.). It doesn't matter whether they are  published or not to be able to use a git reference like a tag.